airbyte-integrations/connectors/source-sentry/source_sentry/manifest.yaml

```diff
@@ -1,105 +1,175 @@
+    paginator:
+      $ref: "#/definitions/paginator"
+    partition_router: []
+  retriever_with_filter:
+    type: SimpleRetriever
+    requester:
+      $ref: "#/definitions/requester"
+    record_selector:
+      $ref: "#/definitions/record_selector"
+      record_filter:
+        condition: "{{ record[parameters['cursor_field']] > stream_state.get(parameters['cursor_field'], '') }}"
+    paginator:
+      $ref: "#/definitions/paginator"
+    partition_router: []
+  incremental_sync:
+    type: DatetimeBasedCursor
+    cursor_field: "{{ parameters['cursor_field'] }}"
+    cursor_datetime_formats:
+      - "%Y-%m-%dT%H:%M:%S.%f%z"
+      - "%Y-%m-%dT%H:%M:%SZ"
+    datetime_format: "%Y-%m-%dT%H:%M:%S.%f%z"
+    start_datetime:
+      type: MinMaxDatetime
+      datetime: "1900-01-01T00:00:00.0Z"
+      datetime_format: "%Y-%m-%dT%H:%M:%S.%fZ"
+    is_data_feed: true
+  base_stream_full_refresh:
     schema_loader:
       $ref: "#/definitions/schema_loader"
     retriever:
       $ref: "#/definitions/retriever"
-      record_selector:
-        $ref: "#/definitions/selector"
+  base_stream_incremental:
+    schema_loader:
+      $ref: "#/definitions/schema_loader"
+    retriever:
+      $ref: "#/definitions/retriever_with_filter"
+    incremental_sync:
+      $ref: "#/definitions/incremental_sync"
+
+  # Stream Events https://docs.sentry.io/api/events/list-a-projects-error-events/
+  events:
+    type: DeclarativeStream
+    $parameters:
+      name: "events"
+      primary_key: "id"
+      path: "projects/{{ config['organization'] }}/{{ config['project'] }}/events/"
+      cursor_field: "dateCreated"
+    retriever:
+      type: SimpleRetriever
       requester:
         $ref: "#/definitions/requester"
-        path: "projects/{{config.organization}}/{{config.project}}/events/"
         request_parameters:
           full: "true"
+      record_selector:
+        $ref: "#/definitions/record_selector"
+        record_filter:
+          condition: "{{ record[parameters['cursor_field']] > stream_state.get(parameters['cursor_field'], '') }}"
       paginator:
         $ref: "#/definitions/paginator"
-  - type: DeclarativeStream
+    incremental_sync:
+      $ref: "#/definitions/incremental_sync"
+
+  # Stream Issues https://docs.sentry.io/api/events/list-a-projects-issues/
+  issues:
+    type: DeclarativeStream
     $parameters:
       name: "issues"
-    primary_key: "id"
-    schema_loader:
-      $ref: "#/definitions/schema_loader"
+      primary_key: "id"
+      path: "projects/{{ config['organization'] }}/{{ config['project'] }}/issues/"
+      cursor_field: "lastSeen"
     retriever:
-      $ref: "#/definitions/retriever"
-      record_selector:
-        $ref: "#/definitions/selector"
+      type: SimpleRetriever
       requester:
         $ref: "#/definitions/requester"
-        path: "projects/{{config.organization}}/{{config.project}}/issues/"
         request_parameters:
           statsPeriod: ""
```

Comment:
I guess this is an issue for sentry because we can't provide `statsPeriod=&` , response is now changed and contains info for stats that we don't have before.

Reply:
This wouldn't be needed given the decision taken https://github.com/airbytehq/airbyte/pull/35755#discussion_r1516371057. I'll still bring the issue to the grooming for our team but this will likely not get prioritized as it is not a blocker anymore